types/viewer.d.ts

```diff
@@ -1,45 +1,44 @@
-import * as THREE from 'three'
-import { CompositeAnimation, WalkAnimation } from './animation'
-import { Animation } from './animation'
-import { PlayerObject } from './model'
-
-interface SkinViewerOptions {
-  domElement: Element
-  animation?: Animation
-  slim?: boolean
-  skinUrl?: string
-  capeUrl?: string
-  width?: number
-  height?: number
+import * as THREE from "three";
+import { Animation } from "./animation";
+import { PlayerObject } from "./model";
+
+export interface SkinViewerOptions {
+	domElement: Node;
```

Comment:
Why use `Node`?

Reply:
`Node` includes all the functions we need to invoke (`appendChild()` and `removeChild()`).